webpack/assets/javascripts/react_app/components/common/Fill/index.js

```diff
@@ -0,0 +1,25 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { selectExtendableComponents } from './FillSelectors';
+
+import * as actions from './FillActions';
+import reducer from './FillReducer';
+
+import Fill from './Fill';
+
+// map state to props
+const mapStateToProps = state => ({
+  components: selectExtendableComponents(state),
```

Comment:
I'm confused, couldn't find any `components` prop the `Fill` using.

Reply:
doesn't needed anymore, thanks !